### PR TITLE
Changelog for v0.12.2

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*
Changelog for patch release v0.12.2
*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/changelog.md [hunk 1]
```diff
@@ -4,15 +4,11 @@ linkTitle: "What's New?"
...
 
-## Unreleased
+## [v0.12.2](https://github.com/aws/eks-anywhere/releases/tag/v0.12.2)
 
 ### Added
 - Add support for Kubernetes 1.24 [#3491  ](https://github.com/aws/eks-anywhere/issues/3491)
 
-### Changed
-
-### Fixed
```
Comment (ahreehong) [inline docs/content/en/docs/reference/changelog.md:14]: - Fix issue with authenticated registry mirror validations

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/content/en/docs/reference/changelog.md [hunk 2]
```diff
@@ -4,14 +4,13 @@ linkTitle: "What's New?"
...
-## Unreleased
+## [v0.12.2](https://github.com/aws/eks-anywhere/releases/tag/v0.12.2)
 
 ### Added
 - Add support for Kubernetes 1.24 [#3491  ](https://github.com/aws/eks-anywhere/issues/3491)
 
-### Changed
-
 ### Fixed
+- Fix authenticated registry mirror validations
```
Comment (maxdrib) [inline docs/content/en/docs/reference/changelog.md:13]: Fix capc bug causing orphaned VM's in slow environments

**Comment by mitalipaygude** [comment]
PR continued here https://github.com/aws/eks-anywhere/pull/4172

**Pull request closed without merging**